src/main/app.js

```diff
@@ -219,217 +373,153 @@ fluid.defaults("gpii.app", {
+ * Changes the keyboard shortcut for opening the GPII app. The previously registered
+ * shortcut for this action (if any) is removed.
+ * @param {Component} shortcutsManager - The `gpii.app.shortcutsManager` instance.
+ * @param {String} shortcut - The new shortcut for opening the GPII app given as an
+ * accelerator string (https://electronjs.org/docs/api/accelerator).
+ * @param {String} oldShortcut - The previously used shortcut for opening the GPII
+ * app given as an accelerator string.
+ * @param {String} eventName - The name of the event which should be triggered when
+ * the new GPII app opening shortcut is activated.
  */
-fluid.defaults("gpii.app.errorHandler", {
-    gradeNames: ["fluid.component"],
-
-    errorsDescriptionMap: {
-        "EADDRINUSE": {
-            title:   "GPII can't start",
-            subhead: "There is another application listening on port the same port",
-            details: "Stop the other running application and try again. If the problem is still present, contact GPII Technical Support.",
-            fatal: true
-        },
-        "EKEYINFAIL": {
-            title:   "Cannot Key In",
-            subhead: "There might be a problem with the user you are trying to use",
-            details: "You can try keying in again. If the problem is still present, contact GPII Technical Support.",
-            btnLabel1: "OK",
-            fatal: false
-        },
-        "ENOCONNECTION": {
-            title:   "No Internet connection",
-            subhead: "There seem to be a problem your Internet connectivity",
-            details: "Have you tried turning it off and on again? If the problem is still present, contact GPII Technical Support.",
-            btnLabel1: "OK",
-            btnLabel2: "Cancel",
-            fatal: false
-        }
-    },
-
-    events: {
-        onFatalError: null
-    },
-
-    listeners: {
-        "onCreate.registerErrorListener": {
-            funcName: "gpii.app.errorHandler.registerErrorListener",
-            args: ["{that}"]
-        },
-        "onDestroy.clearListener": {
-            funcName: "fluid.onUncaughtException.removeListener",
-            args: ["gpii.app.errorHandler"]
-        }
-    },
-
-    // Attach keyIn error listener to the core component
-    distributeOptions: {
-        target: "{flowManager requests stateChangeHandler}.options.listeners.onError",
-        record: "gpii.app.errorHandler.onKeyInError"
-    },
-
-    invokers: {
-        handleUncaughtException: {
-            funcName: "gpii.app.errorHandler.handleUncaughtException",
-            args: [
-                "{that}",
-                "{dialogManager}",
-                "{that}.options.errorsDescriptionMap",
-                "{arguments}.0" // error
-            ]
-        }
+gpii.app.changeGpiiAppShortcut = function (shortcutsManager, shortcut, oldShortcut, eventName) {
+    if (oldShortcut) {
+        shortcutsManager.deregisterGlobalShortcut(oldShortcut);
```

Comment:
This line is untested

Reply:
I have just added some additional tests which cover this.